hasjob/static/js/app.js

```diff
@@ -192,6 +192,25 @@ window.Hasjob.Filters = {
       return '';
     }
   },
+  formatQuery: function(query){
+    return query ? query.replace(/'/g, '') : "";
+  },
+  getCurrentState: function(){
+    return {
+      jobLocations: window.Hasjob.Config.allFilters.job_location_filters,
+      jobTypes: window.Hasjob.Config.allFilters.job_type_filters,
+      jobCategories: window.Hasjob.Config.allFilters.job_category_filters,
+      jobsArchive: window.Hasjob.Config.selectedFilters.archive,
+      selectedLocations: window.Hasjob.Config.selectedFilters.location_names,
+      selectedTypes: window.Hasjob.Config.selectedFilters.types,
+      selectedCategories: window.Hasjob.Config.selectedFilters.categories,
+      selectedQuery: this.formatQuery(window.Hasjob.Config.selectedFilters.query),
+      selectedCurrency: window.Hasjob.Config.selectedFilters.currency,
+      pay: window.Hasjob.Config.selectedFilters.pay,
+      equity: window.Hasjob.Config.selectedFilters.equity,
```

Comment:
Add tags. It's not in the UI yet, but we'll have to find a place to insert. Tags use `make_name`, so they won't have spaces.